app/controllers/api/auth_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::AuthController < ApiController
+  skip_before_action :authenticate_token!
+
+  def create
+    user = User.find_by(email: params[:user][:email])
```

Comment:
- [ ] @Team Great work adding the API in your app, however am getting the error below, when trying to log in the user, which is preventing me from checking if your posts and comments are displayed correctly and user can add new comment
![image](https://github.com/Young-Gaius/Rails-Blog-App/assets/61408860/3f470ae3-38ce-432a-b239-3a9bf1079e4c)

Reply:
- You have done well so far, but even after getting a token I get the error below, please make sure that your application works as expected 
![rgererggr](https://github.com/Young-Gaius/Rails-Blog-App/assets/73607512/86b0f88b-8edf-41a8-9ff7-4de43d8ba3a4)

**OPTIONAL** - To help make your authentication easier, I suggest you use the [deviseJWT](https://enlear.academy/how-to-create-a-rails-6-api-with-devise-jwt-46fa35085e85) to handle authentication for your application. Please fix this 🙏